src/sidebars/star/star.less

```diff
@@ -15,34 +19,40 @@
   outline: 0;
   display: block;
   .confirmation-modal {
+    &.flipped{
+      box-shadow: -2px -2px 2px 0px @dark-background;
+    }
+    box-shadow: -2px 2px 2px 0px @dark-background;
```

Comment:
I'd quite like to see a `box-shadow(@shadow-position, @blur, @colour)` mixin, which could be used from the search.less and mixins.less files too.